cmd/edge-orchestration/main.go

```diff
@@ -70,6 +71,9 @@ const (
 	cipherKeyFilePath = edgeDir + "/user/orchestration_userID.txt"
 	deviceIDFilePath  = edgeDir + "/device/orchestration_deviceID.txt"
 	mnedcServerConfig = edgeDir + "/mnedc/client-config.yaml"
+
+	mqttBrokerURL = "ec2-34-228-146-136.compute-1.amazonaws.com" //**Hardcoded URL and port, will be replaced by the servivce app in incremental commit
+	mqttport      = 1883
```

Comment:
These values are gonna be set via third party through the API exposed to it. This is set just for our testing purpose and since its a incremental push. Once the API is written this will  be removed.

Reply:
You can assign the broker addr for testing with an environment variable like below when launching edge orchestration. 
```
docker run -it -d --privileged --network="host" --name edge-orchestration -e CLOUDSYNC=ec2-34-228-146-136.compute-1.amazonaws.com
```

Reply:
done!!